docs/source/redhat-opens-install.rst

```diff
@@ -314,6 +314,16 @@ On each control node, perform the following steps:
 
        service neutron-server restart
 
+7. Find all the currently running OpenStack Neutron agents and stop them.
+   First, run the following command to find them all::
+
+       neutron agent-list
+
+   Then, for each agent, delete them with the following command, replacing
+   ``<agent-id>`` with the ID of the agent::
+
+       neutron agent-delete <agent-id>
```

Comment:
I think that the agents need to be stopped using systemdctl or upstart's equivalent before you remove them from the DB.  Otherwise the DB will get repopulated as soon as the agent reconnects.